frontend/src/pages/RecipeSearch/RecipeSearch.jsx

```diff
@@ -16,26 +18,39 @@ export default function RecipeSearch() {
 
     async function handleSubmit() {
         try {
-            const result = await axios.get('http://localhost:4000/api/searchbyingredient', {
-                params: {
-                    ingredients: ingredients
-                }
-            });
+            // const result = await axios.get('http://localhost:4000/api/searchbyingredient', {
+            //     params: {
+            //         ingredients: ingredients
+            //     }
+            // });
+
+            //! test
+            let result = { data: searchSample };
+            console.log("result", result.data);
+
+            const recipeInstructions = bulkSample
+
+            let combinedInfo = combine(searchSample, bulkSample)
+            result = { data: combinedInfo }
+            //! test
+
 
             if (result?.data) {
                 setError(false);
                 setRecipeList(result.data);
             }
-            
+
             const recipeIdList = result.data.map(recipe => recipe.id)
 
 
-            const recipeInstructions = await axios.get('http://localhost:4000/api/recipeinformation', {
-                params: {
-                    recipeIdList: recipeIdList
-                }
-            })
-            
+            // const recipeInstructions = await axios.get('http://localhost:4000/api/recipeinformation', {
+            //     params: {
+            //         recipeIdList: recipeIdList
+            //     }
+            // })
+
```

Comment:
I think this should probably be un-commented out so the API call is still accessible in this file.